tutorials/distributed-keras-on-cmle/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Distributed Keras on Google Cloud Machine Learning Engine
+description: Learn how to use distributed Keras on Cloud Machine Learning Engine.
+author: corrieelston
+tags: Data Science, Cloud Machine Learning Engine, Keras, Python
+date_published: 2017-09-28
+---
+
+# Distributed Keras on Google Cloud Machine Learning Engine (CMLE)
+
+This solution connects the dots between a number of technologies with the end result of a production-grade model.
+
+First the technologies:
+
+* [TensorFlow](https://www.tensorflow.org/) (including [tf.contrib.learn](https://www.tensorflow.org/get_started/tflearn)) - TensorFlow is the most popular deep learning library available.  It is functional and performant, but equally importantly, has a vibrant community of users producing educational and reference materials to accelerate your progress towards adding value to your business.
+* [Keras](https://keras.io/) - Keras is a high-level, Python API focused on enabling fast experimentation, with the motivation that moving from idea to result with the least possible delay is the key to good research (and machine learning in general).  It is being integrated into TensorFlow (currently as tf.contrib.keras) but is also available standalone and can run on TensorFlow, CNTK or Theano.
```

Comment:
By looking at the activities below, I see that you implement your model in Keras after an initial estimator-based version. It isn't at all clear why you're doing that. What is Keras adding to the mix in this solution?